### DISCUSSION TITLE
Which CUDA version is required for ONNX Runtime v1.18.0?

### DISCUSSION THREAD

**Discussion by simon-eisenmann-driveblocks** [microsoft/onnxruntime#21480] [EP Q&A] [answered]
Hi,

I am planning to run inference using ONNX Runtime v1.18.0 in a C++ application with CUDA support. Could someone please let me know which CUDA version is compatible with this version of ONNX Runtime?

Thanks!

**Comment by carzh** [comment, marked as answer]
It looks like CUDA 12.x and 11.8 are compatible with 1.18.0. Version compatibility tables can be found [here](https://onnxruntime.ai/docs/execution-providers/CUDA-ExecutionProvider.html)

**Reply by simon-eisenmann-driveblocks** [reply]
Thanks!
